ecl/hqlcpp/hqlttcpp.cpp

```diff
@@ -1287,19 +1299,33 @@ IHqlExpression * HqlGlobalDatasetTransformer::createTransformed(IHqlExpression *
 bool HqlGlobalDatasetTransformer::needToTransform() const
 {
     if (consistencyError)
-    {
-        if (doTrace(traceOptimizations))
-            DBGLOG("HqlGlobalDatasetTransformer: Skipping transformation due to consistency error");
         return false;
-    }
 
+    bool hasCandidate = false;
     ForEachItemIn(i, allGlobals)
```

Comment:
Can there be more than one GLOBAL in a graph that can be transformed via this mechanism? If so, would both be transformed to spills?